packages/shared/src/components/sidebar/Sidebar.spec.tsx

```diff
@@ -129,40 +121,6 @@ it('should not render create my feed button if user has alerts.filter as false',
   expect(createMyFeed).not.toBeInTheDocument();
 });
 
-it('should remove filter alert if the user has filters and opened feed filters', async () => {
```

Comment:
Note: removal of this test case was intentional since the sidebar won't be able to open the "Feed filters" menu with the current changes as it will now be part of the header for `my-feed` page. And in order for you to have a `my-feed` page, you must go through the feed filters onboarding process which has its own test suite that handles this case.

Relevant test suite: `TagsFilter.spec.tsx`